geode-docs/developing/eviction/configuring_data_eviction.html.md.erb

```diff
@@ -38,51 +38,26 @@ Configure data eviction as follows. You do not need to perform these steps in th
 
-**Note:**
-You can also configure Regions using the gfsh command-line interface, however, you cannot configure `eviction-attributes` using gfsh. See [Region Commands](../../tools_modules/gfsh/quick_ref_commands_by_area.html#topic_EF03119A40EE492984F3B6248596E1DD) and [Disk Store Commands](../../tools_modules/gfsh/quick_ref_commands_by_area.html#topic_1ACC91B493EE446E89EC7DBFBBAE00EA).
-
-Examples:
-
-``` pre
-// Create an LRU memory eviction controller with max bytes of 1000 MB
-// Use a custom class for measuring the size of each object in the region 
-<region-attributes refid="REPLICATE"> 
-  <eviction-attributes> 
-    <lru-memory-size maximum="1000" action="overflow-to-disk"> 
-      <class-name>com.myLib.MySizer</class-name> 
-      <parameter name="name"> 
-        <string>Super Sizer</string> 
-      </parameter> 
-    </lru-memory-size> 
-  </eviction-attributes> 
-  </region-attributes>
-```
+**Examples:**
 
-``` pre
-// Create a memory eviction controller on a partitioned region with max bytes of 512 MB
-<region name="demoPR">
-  <region-attributes refid="PARTITION">
-    <partition-attributes local-max-memory="512" total-num-buckets="13"/>
-    <eviction-attributes>
-       <lru-memory-size action="local-destroy"/>
-       <class-name>org.apache.geode.cache.util.ObjectSizer
-       </class-name>
-    </eviction-attributes>
-  </region-attributes>
-</region>
-            
-```
+Create an LRU memory eviction controller with a maximum limit of 1000 MB. Use a custom class for measuring the size of each object in the region:
 
-``` pre
-// Configure a partitioned region for heap LRU eviction. The resource manager controls the limits. 
-<region-attributes refid="PARTITION_HEAP_LRU"> 
-</region-attributes>
 ```
+gfsh>create region --type=REPLICATE --eviction-max-memory=1000 \
+--eviction-action=overflow-to-disk --eviction-object-sizer=com.myLib.MySizer
+```
+
+Create a memory eviction controller on a partitioned region with a maximum entry count of 512:
 
-``` pre
-Region currRegion = cache.createRegionFactory()
-    .setEvictionAttributes(EvictionAttributes.createLRUHeapAttributes(EvictionAction.LOCAL_DESTROY))
-    .create("root");
+```
+gfsh>create region --name=myRegion --type=PARTITION --eviction-entry-count=512 \
+--eviction-action=local-destroy 
 ```
 
+To configure a partitioned region for heap LRU eviction, first configure the resource manager on server startup, then create a region with eviction enabled:
 
+```
+gfsh>start server --name Server1 --eviction-heap-percentage=80
+...
+gfsh>create region --type=PARTITION --eviction-enabled
```

Comment:
The region name must be specified for the create region command.  The start server's command needs an equals sign for specifying the server name.